blog/author/iryna-kheroim/feed/index.xml

```diff
@@ -607,7 +607,7 @@
 
 
 
-<p style="font-size:16px">If you want to learn more about this topic, see a full description of the problems and their solutions using anomaly detection, and check out the detailed step-by-step&nbsp;<a href="/blog/optimizing-solar-parks-efficiency-advanced-anomaly-detection-solutions/">guide here</a>.</p>
+<p style="font-size:16px">If you want to learn more about this topic, see a full description of the problems and their solutions using anomaly detection, and check out the detailed step-by-step&nbsp;<a href="https://thingsboard.io/blog/optimizing-solar-parks-efficiency-advanced-anomaly-detection-solutions/" title="">guide here</a>.</p>
```

Comment:
Why title empty? The title is an optionally defined attribute, please paste advisory information about a link or just remove it.